sdks/java/extensions/zetasketch/build.gradle

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
```

Comment:
If current project depends on other project, it's recommended to declare 'evaluationDependsOn'  first like: https://github.com/apache/beam/blob/master/runners/direct-java/build.gradle#L50

Reply:
Interesting. I looked into the usage of `evaluationDependsOn`. From the comments in the link you shared
https://github.com/apache/beam/blob/master/runners/direct-java/build.gradle#L46-L48
it says it is only needed because in that script `sourceSets.test.output` of another project is directly referenced
https://github.com/apache/beam/blob/master/runners/direct-java/build.gradle#L102-L104
And I searched throughout the codebase and it seems to be the only reason that people adding `evaluationDependsOn` to their build script.

However, direct reference of `sourceSets.test.output` is discouraged. Quote from https://discuss.gradle.org/t/evaluationdependson-annoyance/21783/2:
>While you can reference tasks from other projects, it really should be a last resort. Using evaluationDependsOn is a bit of a smell. Dependencies, configurations, and artifacts are a better model than directly accessing outputs of a task from another project.

So I would prefer not to add it to this script because we are not doing the direct referencing here.